metricbeat/metricbeat.reference.yml

```diff
@@ -275,6 +275,7 @@ metricbeat.modules:
     #- index
     #- index_recovery
     #- index_summary
+    #- ingest
```

Comment:
Slightly surprised by the name. Is this all of ingest or just ingest pipelines?

Reply:
After discussion, I agree. Going to rename this to ingest pipelines.